### PR TITLE
add theme A-Dearmsdan

### PR DESCRIPTION
**Pull request by ZHD99**
## Check List

**Please read and check followings before submitting a PR.**

- [x] I want to publish my theme on Hexo official website.
    - [x] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [x] `name` is unique.
    - [x] `link` URL is correct.
    - [x] `preview` URL is correct.
    - [x] `preview` URL web site is rendered correctly.
    - [x] Add a screenshot to `source/themes/screenshots`.
    - [x] Screenshot filename is same as value of `name`.
    - [x] Screenshot size is `800 * 500`.
    - [x] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)
    - Languages:
    - [ ] `en` English
    - [ ] `ko` Korean
    - [ ] `pt-br` Brazilian Portuguese
    - [ ] `ru` Russian
    - [ ] `th` Thai
    - [ ] `zh-cn` simplified Chinese
    - [ ] `zh-tw` traditional Chinese

### CODE REVIEW

**Comment by ZHD99** [comment]
... sorry ，I can't handle it very well，git 我不太会玩

**Comment by ZHD99** [comment]
在之前的一个空格影响了 netlify- 部署，我以为是图片的大小问题，我把图片压缩大小重新上传了一次，然后发现是theme 的空格问题

File: source/_data/themes.yml [hunk 1]
```diff
@@ -3135,3 +3135,14 @@
     - simple
     - fast
     - latest-comment
+- name: A-Dearmsdan
```
Comment (SukkaW) [inline source/_data/themes.yml:3138]:
```suggestion
- name: Dearmsdan
```

**Comment by stevenjoezhang** [comment]
The link of the preview image is incorrect. Please rename it.

**Pull request merged**